### PR TITLE
DOCS-7866 added Simple rule creator for CSM Threats

### PR DESCRIPTION
**Pull request by michaelcretzman**
DOCS-7866

### What does this PR do? What is the motivation?
Updates CSM custom rule creation with new Simple rule creator feature.

### Merge instructions

Please DO NOT merge after reviewing. Still waiting on whether this is beta or GA.

### Additional notes
Updated menu title for another topic:
- Creating Custom Agent Rules > Creating Agent rule expressions

### CODE REVIEW

**Comment by aliciascott** [comment]
Editorial card: https://datadoghq.atlassian.net/browse/DOCS-7922

File: content/en/security/threats/workload_security_rules/custom_rules.md [hunk 1]
```diff
@@ -16,13 +16,64 @@ further_reading:
...
+
+1. In [Agent Configuration][4] or [Threat Detection Rules][3], select **New Rule**, and then select **Simple rule creator**.
+2. Define the detection.
+   - To detect unauthorized changes to files, select **File integrity monitoring (FIM)**.
+   - To track and analyze system software processes for malicious behavior or policy violations, select **Process activity monitoring**.
+   - Enter the file/process names or paths to monitor. 
+3. Specify more conditions. Enter any arguments to add to the threat rule expression. For example, the argument `foo` is added as `process.argv in ["foo"]`.
+4. Set severity and notification lists. 
+   - Select the severity for the signal generated when this threat is detected. 
+   - Select notification lists to notify when a signa is generated.
```
Comment (jhgilbert) [inline content/en/security/threats/workload_security_rules/custom_rules.md:57]:
```suggestion
   - Select notification lists to notify when a signal is generated.
```

File: content/en/security/threats/workload_security_rules/custom_rules.md [hunk 2]
```diff
@@ -16,13 +16,64 @@ further_reading:
...
+5. Add rule name and description.
```
Comment (jhgilbert) [inline content/en/security/threats/workload_security_rules/custom_rules.md:58]:
```suggestion
5. Add the rule name and description.
```

File: content/en/security/threats/workload_security_rules/custom_rules.md [hunk 3]
```diff
@@ -16,13 +16,64 @@ further_reading:
...
+   
+   Here's an example of a new FIM rule, including the expressions generated for each rule.
+
+    {{< img src="/security/csm/csm_threats_simple_rule_creator.png" alt="Simple rule creator example" style="width:100%;" >}}
+
+6. Select **Create `<<NUMBER>>` Rules**.
```
Comment (jhgilbert) [inline content/en/security/threats/workload_security_rules/custom_rules.md:64]:
```suggestion
6. Select **Create `<NUMBER>` Rules**.
```
This came up in backroom today, but a simpler convention would be **Create _N_ Rules**, if you prefer that.

File: content/en/security/threats/workload_security_rules/custom_rules.md [hunk 4]
```diff
@@ -16,13 +16,64 @@ further_reading:
...
+7. In **Generate Rules**, select **Confirm**. The rules are generated.
+8. Select **Finish**. The [Agent Configuration][3] page displays the new rules.
+9. In [Agent Configuration][3], select **Deploy Agent Policy**.
+
+
+## Create a custom Agent rule
+
+You can create an individual custom Agent rule, deploy it as a [new Agent policy](#deploy-the-policy-in-your-environment), and then reference it in a [custom detection rule](#create-a-custom-detection-rule).
```
Comment (jhgilbert) [inline content/en/security/threats/workload_security_rules/custom_rules.md:72]:
```suggestion
You can create an individual custom Agent rule, deploy it as a [new Agent policy](#deploy-the-policy-in-your-environment), and reference it in a [custom detection rule](#create-a-custom-detection-rule).
```

File: content/en/security/threats/workload_security_rules/custom_rules.md [hunk 5]
```diff
@@ -16,13 +16,64 @@ further_reading:
...
-In addition to the default rules, you can write custom Agent and detection rules. Custom Agent rules are deployed to the Agent in a custom policy separate from the default one. The custom policy contains custom Agent rules as well as [default rules that have been disabled](#disable-default-agent-rules).
+This topic explains how to create custom Datadog Agent and detection rules for [CMS Threats][8].
 
-### Define the agent rule
+In addition to the out of the box (OOTB) [default Agent and detection rules][7], you can write custom Agent and detection rules. Custom rules help to detect events Datadog is not detecting with its OOTB rules.
 
-1. On the [**Agent Configuration**][4] page, click **New Rule**.
+## Custom detection rules summary
+
+Understanding the dependency detection rules have on Agent rules is important when you want to use custom rules. Detection rules are composed of existing, deployed Agent rules and additional expression parameters.
```
Comment (jhgilbert) [inline content/en/security/threats/workload_security_rules/custom_rules.md:25]:
```suggestion
Custom detection rules depend on Agent rules. They are composed of existing, deployed Agent rules and additional expression parameters. 
```
I found the copy as written a little hard to parse -- will this work? I wasn't sure I captured your meaning accurately.

**Review by jhgilbert — APPROVED** [review]
Approved with some suggestions, thanks!

**Comment by paragbaxi** [comment]
approved
comments below

> Using this tool is faster than the advanced method of creating the Agent and detection rules separately.

love it

---

> In Agent Configuration or [Threat Detection Rules](https://app.datadoghq.com/security/configuration/workload/rules)

you are on top of it

---

> To detect unauthorized changes to files, select File integrity monitoring (FIM).
To track and analyze system software processes for malicious behavior or policy violations, select Process activity monitoring.
Enter the file/process names or paths to monitor.

this is good. I wonder if clarifying that a rule can only be a fim or process is needed
something like
> To monitor your resource effectively, you have the option to choose between two detection types:
To detect unauthorized changes to files, select File integrity monitoring (FIM).
To track and analyze system software processes for malicious behavior or policy violations, select Process activity monitoring.
Enter the file/process names or paths to monitor.

could just be an unnecessary change
i'll leave it up to you

---

> 6. Select Create <<NUMBER>> Rules.

should we clarify that NUMBER is populated by the system, or would the user already get that with the <<>>.
again, it's understandable as is

File: content/en/security/threats/workload_security_rules/custom_rules.md [hunk 6]
```diff
@@ -16,13 +16,64 @@ further_reading:
...
+## Create the custom Agent and detection rules together
+
+The **Simple rule creator** option helps you create the Agent and dependent detection rules together, and ensures that the Agent rule is referenced in the detection rules. Using this tool is faster than the advanced method of creating the Agent and detection rules separately.
+
+As you define the rules using this tool, the threat expressions generated for these rules are displayed in the tool.
+
+To use the simple rule creator:
+
+1. In [Agent Configuration][4] or [Threat Detection Rules][3], select **New Rule**, and then select **Simple rule creator**.
+2. Define the detection.
```
Comment (michaelcretzman) [inline content/en/security/threats/workload_security_rules/custom_rules.md:50]:
```suggestion
2. Define the detection. To monitor your resource effectively, you have the following detection type options:
```

File: content/en/security/threats/workload_security_rules/custom_rules.md [hunk 7]
```diff
@@ -16,13 +16,64 @@ further_reading:
...
+4. Set severity and notification lists. 
+   - Select the severity for the signal generated when this threat is detected. 
+   - Select notification lists to notify when a signal is generated.
+5. Add the rule name and description.
+   
+   Here's an example of a new FIM rule, including the expressions generated for each rule.
+
+    {{< img src="/security/csm/csm_threats_simple_rule_creator.png" alt="Simple rule creator example" style="width:100%;" >}}
+
+6. Select **Create `<NUMBER>` Rules**.
```
Comment (michaelcretzman) [inline content/en/security/threats/workload_security_rules/custom_rules.md:64]:
```suggestion
6. Select **Create _N_ Rules**.
```
